File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.cc
#include <stdio.h>

int main(void) {
	int arrA[100][100], arrB[100][100];
	int n, m;
	scanf("%d %d", &n, &m);
	for (int i = 0; i < n; i++)
		for (int j = 0; j < m; j++)
			scanf("%d", &arrA[i][j]);
	for (int i = 0; i < n; i++)
		for (int j = 0; j < m; j++)
			scanf("%d", &arrB[i][j]);
	for (int i = 0; i < n; i++) {
		for (int j = 0; j < m; j++)
			printf("%d ", arrA[i][j] + arrB[i][j]);
		printf("\n");
	}
	return 0;
}

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        int[][] arrA = new int[n][m];
        int[][] arrB = new int[n][m];
        for(int i = 0; i < n; i++)
            for(int j = 0; j < m; j++)
                arrA[i][j] = sc.nextInt();
        for(int i = 0; i < n; i++)
            for(int j = 0; j < m; j++)
                arrB[i][j] = sc.nextInt();
        for(int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++)
                System.out.printf("%d ", arrA[i][j] + arrB[i][j]);
            System.out.println();
        }
    }
}